src/components/StrategiesListTable/StrategiesList.columns.js

```diff
@@ -27,7 +28,7 @@ const activeStrategiesColumns = (t, getMarketPair) => [
     headerStyle: STYLES.flexStart,
     width: 100,
     flexGrow: 1,
-    cellRenderer: ({ rowData = {} }) => defaultCellRenderer(getMarketPair(rowData.strategyOptions.symbol)),
+    cellRenderer: ({ rowData = {} }) => defaultCellRenderer(_isEmpty(rowData.strategyOptions?.symbol) ? 'N/A' : getMarketPair(rowData.strategyOptions.symbol)),
```

Comment:
could you explain this changes?

Reply:
the app was crashing for me because on older `executed strategies` the `symbol` is `undefined`.
added this fix to prevent the crash for now and in future if somehow it happens again.

Reply:
@Blaumaus as executed strategies were not in production - we have to delete related file `executed-strategies.json` after each changes.

Reply:
I think it is better to change `getMarketPair` fn, and return `'N/A'` from this fn. Because we still have unresolved issue with missing tickers (like LUNA)